lib/puppet/property/ensure.rb

```diff
@@ -54,7 +54,7 @@ def change_to_s(currentvalue, newvalue)
       elsif newvalue == :absent
         return _("removed")
       else
-        return _("%{value0} changed '%{value1}' to '%{value2}'") % { value0: self.name, value1: self.is_to_s(currentvalue), value2: self.should_to_s(newvalue) }
+        return _('%{value0} changed %{value1} to %{value2}') % { value0: self.name, value1: self.is_to_s(currentvalue), value2: self.should_to_s(newvalue) }
```

Comment:
We can. But I suggest doing that in another ticket where we also address the fact that words like "removed"  3 lines up shouldn't be translated. These are terms used in the manifest and it would be very confusing if the messages doesn't use those terms. I say that under the assumption that manifests will still be written using English terminology.